zigpy/util.py

```diff
@@ -72,20 +73,27 @@ class LocalLogMixin:
     def log(self, lvl: int, msg: str, *args, **kwargs):  # pragma: no cover
         pass
 
+    def _log(self, lvl: int, msg: str, *args, **kwargs):
+        if sys.version_info >= (3, 8):
+            # We have to exclude log, _log, and info
+            return self.log(lvl, msg, *args, stacklevel=4, **kwargs)
+        else:
```

Comment:
can you remove else and just do return here?

Reply:
Done